extensions/dacpac/src/wizard/api/basePage.ts

```diff
@@ -67,19 +67,14 @@ export abstract class BasePage {
 				}
 			}
 
-			let db = c.options.databaseDisplayName;
 			let usr = c.options.user;
 			let srv = c.options.server;
 
-			if (!db) {
-				db = localize('basePage.defaultDb', '<default>');
-			}
-
 			if (!usr) {
 				usr = localize('basePage.defaultUser', 'default');
 			}
 
-			let finalName = `${srv}, ${db} (${usr})`;
+			let finalName = `${srv} (${usr})`;
```

Comment:
I think we have the same bug in the flat file import wizards.  it might be a quick-win to resolve this same bug in there as well.

Reply:
sure I removed it there too. @kenvanhyning just fyi I removed the database name in the server dropdown in the flat file import wizard
![image](https://user-images.githubusercontent.com/31145923/60038231-9f9ad380-9668-11e9-8845-7fec59d846c3.png)